javascripts/googleAnalyticsImporter.js

```diff
@@ -39,11 +41,19 @@ function hidePendingNotification(){
 }
 
 
-function displayPendingNotification(availableDate){
+function displayPendingNotification(availableDate, messageType) {
+  if (!$('#notificationContainer').length) {
```

Comment:
Added this since the twig was missing here - https://github.com/matomo-org/matomo/pull/20123 and we would need a Core upgrade in plugin for this to work.
To solve this issue we added this line of code